osgtest/tests/test_60_singularity.py

```diff
@@ -0,0 +1,53 @@
+import os
+import osgtest.library.core as core
+import osgtest.library.files as files
+import osgtest.library.osgunittest as osgunittest
+import tempfile
+
+class TestSingularity(osgunittest.OSGTestCase):
+
+    __check_path = '/cvmfs/cms.cern.ch/cmsset_default.sh'
+    __cvmfs_image = '/cvmfs/singularity.opensciencegrid.org/opensciencegrid/osg-wn:3.3-el6'
+
+    def mountSingularityCVMFSRepo(self, repo):
+        command = ('mkdir', '-p', /cvmfs + repo)
+        status, stdout, stderr = core.system(command, False)
+        if status != 0:
+            self.fail("failed to mkdir /cvmfs/%s" % repo)
+
+        command = ('mount', '-t', 'cvmfs', 'repo', '/cvmfs/' + repo)
+        status, stdout, stderr = core.system(command, False)
+        if status != 0:
+            self.fail("failed to mount: %s, error: %s" % (repo,stdout))
```

Comment:
Both of these commands and associated status checking look like good candidates to be replaced with `core.check_system()`